kolibri/plugins/management/assets/src/device_management/views/manage-content-page/task-progress.vue

```diff
@@ -74,6 +75,17 @@
         required: true,
       },
       id: RequiredString,
+      cancellable: {
```

Comment:
This needs to be defaulted to `true` for the import/export case (throws out prop validation warnings).

Reply:
Nevermind, I think the issue is that this API-to-state mapper actually omits `cancellable`

```
function transformTasks(tasks) {
  return tasks.map(task => ({
    id: task.id,
    type: task.type,
    status: task.status,
    metadata: task.metadata,
    percentage: task.percentage,
  }));
}

```

http://github.com/rtibbles/kolibri/blob/d7d88f3b3cec079e6772845c68c55fbb2e989f1a/kolibri/plugins/management/assets/src/device_management/state/actions/taskActions.js#L13

Reply:
Ahh, yes, should change that.